src/connection-editor/page-wifi-security.c

```diff
@@ -407,6 +412,19 @@ finish_setup (CEPageWifiSecurity *self, gpointer user_data)
 		}
 	}
 
+	if (security_valid (NMU_SEC_SAE, mode)) {
```

Comment:
It seems to me security_valid () always returns FALSE for SAE in non-ap mode... is that on purpose?

Reply:
Sort of. NetworkManager's 0ff1cb556ca9b8a77a0b5b25522876c889b3835d fixes this. Instead of cheating a bit more, I'd prefer to pick that commit to NM's 1.20 branch.